File: frontend/src/services/AccountService.js
import api from './api'

const AccountService = {
  createAccount: (data) => api.post('/api/accounts', data),
  getAccounts: () => api.get('/api/accounts'),
  getAccountById: (id) => api.get(`/api/accounts/${id}`),
  updateAccount: (id, data) => api.put(`/api/accounts/${id}`, data),
}

export default AccountService

File: backend/backup/service/AccountService.java
package com.bankingsystem.service;

import com.bankingsystem.dto.*;
import java.util.List;

public interface AccountService {
    AccountResponse createAccount(String email, AccountRequest request);
    List<AccountResponse> getAccounts(String email);
    AccountResponse getAccountById(String email, Long id);
    AccountResponse updateAccount(String email, Long id, AccountRequest request);
}
